sdk/digitaltwins/digitaltwins/swagger/README.md

```diff
@@ -0,0 +1,34 @@
+# KeyVault Keys Swagger Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+typescript:
+  package-name: "@azure/digitaltwins"
+  title: GeneratedClient
+  description: Digitaltwins Client
+use-extension:
+  "@autorest/typescript": "6.0.0-dev.20200727.1"
+azure-arm: true
+generate-metadata: false
+add-credentials: true
```

Comment:
```suggestion
add-credentials: false
```

Reply:
It is needed to be true, otherwise the generated code will not provide input for "credentials" in the constructor.

---
In reply to: [470205528](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r470205528) [](ancestors = 470205528)

Reply:
The generated client doesn't need to take credentials, you set up the authentication policy with `bearerTokenAuthenticationPolicy` and passed it through `createPipelineFromOptions` - so the `ServiceClient` base class doesn't need to do auth for you. You can just set this to false and not pass the credential and it will work fine.

Reply:
The base class taking credentials is a compat thing for track1 clients that didn't make a pipeline manually.